user_manual/docs/firmware.md

```diff
@@ -5,18 +5,39 @@ Updating Firmware
 
 # Overview {#update-overview}
 
-Discuss boot loader vs application firmware.
+The **Firmware** is the code that runs on the board.
+
+The **Bootloader** is a mode that allows the firmware to be written to the board
 
 # Application Firmware Update {#update-steps}
 
 ## Enter Bootloader Mode {#update-bootloader}
+To enter the Bootloader mode press and hold the button on the board when powering up. If you are successful the top LED will be solid and the bottom three LEDS will flash together
 
 ## Flash Firmware {#update-flash}
+Download the latest firmware package from [firmware] and note its location
+### On Linux {#update-flash-linux}
+Download the **dfu-util** application using
+
+`sudo apt-get install dfu-util`
+
+To update the firmware run the following command replacing "firmware.dfu" with the file path to the firmware you downloaded earlier
+
+`sudo dfu-util -a 0 -R -D firmware.dfu`
+
+### On Mac {#update-flash-mac}
+
+### On Windows {#update-flash-windows}
+
+# Bootloader Update {#bootloader-update}
+In order to update the bootloader you will need a special set of hardware and software that can interface with the PIC32 microcontroller. 
 
-# Bootloader Update
+## Hardware Required {#bootloader-update-hardware}
+You will need a PicKit 3 programmer or similar
 
-## Hardware Required
+## Software Required {#bootloader-update-software}
+MPLAB IPE
 
-## Software Required
+## Steps {#bootloader-update-steps}
 
-## Steps
+[firmware]:	http://dl.openlighting.org	"firmware"
```

Comment:
So @hezmand this can be https://github.com/OpenLightingProject/ja-rule/releases for now.